dotnet/src/webdriver/WebDriver.cs

```diff
@@ -612,7 +610,7 @@ protected virtual Response Execute(string driverCommandToExecute,
         /// <returns>A <see cref="Response"/> containing information about the success or failure of the command and any data returned by the command.</returns>
         protected virtual async Task<Response> ExecuteAsync(string driverCommandToExecute, Dictionary<string, object> parameters)
         {
-            Command commandToExecute = new Command(this.sessionId, driverCommandToExecute, parameters);
+            Command commandToExecute = new Command(this.SessionId, driverCommandToExecute, parameters);
```

Comment:
I remember it is diff topic, I propose to not use `this` for properties. I belive everybody thumbs up.

Reply:
I adjusted.